ext/RecursiveArrayToolsReverseDiffExt.jl

```diff
@@ -0,0 +1,15 @@
+module RecursiveArrayToolsReverseDiffExt
+
+function RecursiveArrayTools.recursivecopy!(b::AbstractArray{T, N},
+                                            a::AbstractArray{T2, N}) where {
+                                                                            T <:
+                                                                            Tracker.TrackedArray,
+                                                                            T2 <:
```

Comment:
`Tracker.TrackedArray` probably isn't in scope?

Reply:
lol...